macros

```diff
@@ -27,10 +27,10 @@
 %_binary_filedigest_algorithm 8
 
 # Use XZ compression for source payloads
-%_source_payload w6T.xzdio
+%_source_payload w19.zstdio
 
 # Use XZ compression for binary payloads
```

Comment:
Maybe change comment as well?

Reply:
Good find, that was a bit too hasty :) Fixed now